src/pages/mesh/advanced/caching/index.md

```diff
@@ -147,6 +182,22 @@ Example 2
 
   - no-store
 
+Example 3
+
+Public and private headers are mutually exclusive, since `private` is more restrictive, API Mesh selects the values associated with the `private` header.
+
+- Source 1 response headers
+
+  - public, max-age=30, s-maxage=600
+
+- Source 2 response headers
+
+  - private, max-age=60
+
+- Combined HTTP response headers
+
+  - private, max-age=30
+
 ### In the mesh configuration file
```

Comment:
Can we rename this section to emphasize the objective? 

Something like `Overriding the cache-control in your mesh`